swarm/api/http/response_test.go

```diff
@@ -40,7 +40,7 @@ func TestError(t *testing.T) {
 		t.Fatalf("Request failed: %v", err)
 	}
 	defer resp.Body.Close()
-	respbody, err = ioutil.ReadAll(resp.Body)
```

Comment:
I would be slightly happier if there would be check for `ioutil.ReadAll()`,  but not checking is fine.

Reply:
i vote for checking the error too, on the cases below too